app/models/fee_calculation.rb

```diff
@@ -11,20 +11,19 @@ class FeeCalculation < ApplicationRecord
 
   class << self
     def from_odp_data(odp_data)
+      return unless odp_data.present? && odp_data.is_a?(Hash)
```

Comment:
Is there ever a case where some of the fee calculation fields in the hash might be missing or malformed that we may need to put in an additional check for?

Reply:
I _think_ that that would probably be handled by the schema validation at API level. It's theoretically possible that they send some kind of technically-valid data that breaks the assumptions made here but at that point I'd be totally guessing about what kind of validations were necessary.